### DISCUSSION TITLE
setPixel and XY policy change

### DISCUSSION THREAD

**Discussion by davepl** [PlummersSoftwareLLC/NightDriverStrip#385] [Announcements]
By popular request, I am removing the exception when you try to draw out of bounds; it will now produce debug logging when you try to, and we should ideally fix cases where it happens, but for now, with my upcoming checking drawing out of bounds will no longer trigger an exception.

**Comment by robertlipe** [comment]
I endorse this policy. Thank you for opening up on this and for announcing
it.
